routers/classes.py

```diff
@@ -1,6 +1,6 @@
 from uuid import UUID
 
-from fastapi import APIRouter, Depends, status, HTTPException
+from fastapi import APIRouter, Depends, status, HTTPException, Patch
 from sqlalchemy.exc import IntegrityError
```

Comment:
Was there a reason why Patch was imported?

Reply:
It's not necessarily required in the endpoint, sorry for the overlook

Reply:
It's cool. I was only asking. Were you able to set up the project locally. It makes sense to at least try the implementation you're send in since there's no test for it at the moment.